Marlin/src/lcd/tft/tft_image.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "tft_image.h"
+#include "cstddef"
```

Comment:
This include is breaking arduino builds on my build actions.

https://github.com/jeffeb3/MarlinBuilder/runs/927540216?check_suite_focus=true#step:10:781

```
/home/jeffe/Downloads/arduino-1.8.13/hardware/tools/avr/bin/avr-g++ -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -Wno-error=narrowing -flto -w -x c++ -E -CC -mmcu=atmega2560 -DF_CPU=16000000L -DARDUINO=10813 -DARDUINO_AVR_RAMBO -DARDUINO_ARCH_AVR -I/home/jeffe/Downloads/arduino-1.8.13/hardware/arduino/avr/cores/arduino -I/home/jeffe/.arduino15/packages/rambo/hardware/avr/1.0.1/variants/rambo /tmp/arduino_build_926809/sketch/src/lcd/tft/tft_image.cpp -o /dev/null -DARDUINO_LIB_DISCOVERY_PHASE
Alternatives for cstddef: []
ResolveLibrary(cstddef)
  -> candidates: []
/tmp/arduino_build_926809/sketch/src/lcd/tft/tft_image.cpp:24:10: fatal error: cstddef: No such file or directory
 #include "cstddef"
          ^~~~~~~~~
compilation terminated.
exit status 1
Error compiling for board RAMBo.
```

It is building in platformio. But not with arduino.

I'm still trying to figure out where my repo fits into all of this, and the best way to get tests upstream. But I figured I should point this out now, while this merge is fresh.